ibm_cloud_sdk_core/authenticators/authenticator.py

```diff
@@ -17,16 +17,20 @@
 from abc import ABC, abstractmethod
 
 class Authenticator(ABC):
+    """This interface defines the common methods and constants associated with an Authenticator implementation.
```

Comment:
I think you can close this string in the same line

Reply:
👍